### PR TITLE
{bio,lib}[GCC/12.3.0] Transrate v1.0.3, crb-blast v0.6.9, colorize v0.7.7, yell v2.2.2

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)
Needs:
- https://github.com/easybuilders/easybuild-easyblocks/pull/3381

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 3381"

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/3381
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3158.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/3e166b117e4b021e5a13dd3525235e11 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3
EB_ARGS="--include-easyblocks-from-pr 3381"

**Comment by boegel** [comment]
Hmm, sanity check on `jsc-zen3` fails with:
```
`terminal_columns': undefined method `winsize' for nil (NoMethodError)
```

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/c/colorize/colorize-0.7.7-GCC-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+source_urls = ['https://rubygems.org/downloads/']
+sources = ['%(name)s-%(version)s.gem']
+checksums = ['d6ab95a5fcdea3c36c3327d38c1e79e2950ee1788506d8489ae35db330937a99']
+
+gem_file = '%(name)s-%(version)s.gem'
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
```
Comment (boegel) [inline easybuild/easyconfigs/c/colorize/colorize-0.7.7-GCC-12.3.0.eb:21]: `binutils` build dependency is not needed, it's already available when using `GCC` toolchain (not when using `GCCcore` as toolchain)

```suggestion
```

File: easybuild/easyconfigs/c/crb-blast/crb-blast-0.6.9-GCC-12.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+homepage = 'https://github.com/cboursnell/crb-blast'
+description = """Conditional Reciprocal Best BLAST - high confidence ortholog assignment."""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
```
Comment (boegel) [inline easybuild/easyconfigs/c/crb-blast/crb-blast-0.6.9-GCC-12.3.0.eb:14]:
```suggestion
```

File: easybuild/easyconfigs/t/Transrate/Transrate-1.0.3-GCC-12.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+patches = ['Transrate-1.0.3_undefined_method_fix.patch']
+checksums = [
+    {'v1.0.3.tar.gz': '2ccb101cfab5a33586ea9e62af2b2f14caf6bc016724d1fef796b427e39fe100'},
+    {'Transrate-1.0.3_undefined_method_fix.patch': 'ead5e51318d6d810fb11b783d517ea38648f62ae2bdd5f3f4dac7baa9ae94d95'},
+]
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
```
Comment (boegel) [inline easybuild/easyconfigs/t/Transrate/Transrate-1.0.3-GCC-12.3.0.eb:25]:
```suggestion
```

File: easybuild/easyconfigs/y/yell/yell-2.2.2-GCC-12.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+source_urls = ['https://rubygems.org/downloads/']
+sources = ['%(name)s-%(version)s.gem']
+checksums = ['1d166f3cc3b6dc49a59778ea7156ed6d8de794c15106d48ffd6cbb061b9b26bc']
+
+gem_file = '%(name)s-%(version)s.gem'
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
```
Comment (boegel) [inline easybuild/easyconfigs/y/yell/yell-2.2.2-GCC-12.3.0.eb:20]:
```suggestion
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3153.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/a67f90492a1eedce73c3ba9e14270f2f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**